### DISCUSSION TITLE
[@types/rtlcss] doesn't represent types of parameters to hook functions

### DISCUSSION THREAD

**Discussion by catrope** [DefinitelyTyped/DefinitelyTyped#59401] [Issues with a @types package]
The hook functions (`pre` and `post` in the `HookOptions` type) are called with two parameters, one with the `Root` type from postcss and one with the `Postcss` type, but these functions are represented in the type definitions as `() => void`.

[a comment by an automated account is not shown]

**Comment by peterblazejewicz** [comment]
fyi: #59427
